cartography/intel/aws/__init__.py

```diff
@@ -53,7 +54,7 @@ def _sync_one_account(
     current_aws_account_id: str,
     update_tag: int,
     common_job_parameters: Dict[str, Any],
-    regions: list[str] | None = None,
+    regions: Union[List[str], None] = None,
```

Comment:
Use the python 3.10 typing style of

`list[str] | None` instead of `Union[List[str], None]`. 

Older code uses the older style, but newer code should use the newer one whereever possible